bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -52,9 +52,6 @@
 data-try-now="{{ _('Try it now') }}"
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
This page uses the familynav not tabzilla which is why it uses site_css to overwrite. Will leave as is and we can tackle replacing familynav implementation with tabzilla in another PR.